packit_service/worker/whitelist.py

```diff
@@ -146,6 +161,22 @@ def is_approved(self, account_name: str) -> bool:
         :param account_name:
         :return:
         """
+
+        # Postgres
+        if DBWhitelist.get_account(account_name) is not None:
+            logger.info("Whitelisted account found in Postgres.")
+            return True
+
+            # Can also do the following like in redis but seems pointless in this case ??
+
+            # db_status = DBWhitelist.get_account(account_name).status
+            # s = WhitelistStatus(db_status)
+            # return (
+            #     s == WhitelistStatus.approved_automatically
+            #     or s == WhitelistStatus.approved_manually
+            # )
```

Comment:
yeah, no need to

Reply:
I actually forgot about accounts that will exist in db, but will have waiting status. So I guess this wasn't pointless after all